pkg/controller/controller.go

```diff
@@ -369,6 +384,7 @@ func (c *Controller) handleConfig(config *rbacsyncv1alpha.RBACSyncConfig) error
 			c.recorder.Eventf(config, corev1.EventTypeWarning,
 				EventReasonBindingError, "%v/%v has no members for group %v",
 				config.Namespace, config.Name, binding.Group)
+			metrics.RBACSyncConfigStatus.WithLabelValues(MetricsRBACSyncConfig, EventReasonBindingError).Inc()
```

Comment:
Are we certain that this should be an error? Could a group legitimately be empty?

Reply:
Maybe we need a EventReasonBindingWarning?

Reply:
sounds good, will update